ui/ducks/send/send.js

```diff
@@ -1061,52 +1068,57 @@ const slice = createSlice({
       // input field
       state.recipient.userInput = action.payload;
     },
-    validateRecipientUserInput: (state, action) => {
+    updateRecipientWarning: (state, action) => {
+      state.recipient.warning = action.payload;
+    },
+    validateRecipientAddress: (state, action) => {
       const { asset, recipient } = state;
 
       if (
         recipient.mode === RECIPIENT_SEARCH_MODES.MY_ACCOUNTS ||
-        recipient.userInput === '' ||
-        recipient.userInput === null
+        recipient.address === '' ||
```

Comment:
can this ever be undefined? If so we could just get away with `if(Boolean(recipient.address )`

Reply:
I like this idea!